external-crates/move/crates/move-compiler/src/typing/ast.rs

```diff
@@ -378,6 +378,22 @@ pub fn pat(ty: Type, pat: UnannotatedPat) -> MatchPattern {
     MatchPattern { ty, pat }
 }
 
+impl ModuleCall {
```

Comment:
Is it used somewhere already or it's an addition that will be used in one of the following PRs?

Reply:
Following PRs